packages/tests/src/components/bd-element.ts

```diff
@@ -0,0 +1,9 @@
+export default class BigDesignElement {
+  selector: string
+  browser: WebdriverIO.BrowserObject
+
+  constructor(selector: string, browser: WebdriverIO.BrowserObject) {
```

Comment:
Typescript hint:

Doing `constructor(foo: string) { this.foo = foo }` is equivalent to `constructor(private readonly foo: string) {}`

So in your case you should be able to get away with doing just `constructor(public selector: string, public browser: WebdriverIO.BrowserObject)` and delete the `this.selector = selector` and `this.browser = browser` as well as the class declarations `selector: string` and `browser: WebdriverIO.BrowserObject`

If you want to futher play around with any things that TS offers I suggest the playground: http://www.typescriptlang.org/play/

You can see the compiled, human readable JS right there

Reply:
✅